spec/system/reports_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+RSpec.describe 'Reports' do
+  let!(:user) { create(:user, name: 'ケン') }
+
+  before do
+    login_as(user)
+  end
+
+  describe 'Report Index' do
+    let!(:report) { create(:report, user: user, title: 'キウイ本を読んだ', content: 'ちょうど良い難易度でした', created_at: Time.zone.local(2023, 8, 30)) }
+
+    it 'displays elements on report index page' do
+      visit reports_path
+
+      expect(page).to have_content('日報の一覧')
+      within 'div#report_1' do
```

Comment:
```suggestion
      within "#report_#{report.id}" do
```

id(`#`)で要素を指定するときはdivタグは不要です。

あと、`_1`は動的に変わるので変数の値を参照しましょう。

Reply:
修正しました